src/components/SearchBar.vue

```diff
@@ -0,0 +1,37 @@
+<template>
```

Comment:
Same first comment about separation responsibility

Reply:
Okay. Gonna do it.